docs/vexim-acl-check-content.conf

```diff
@@ -1,44 +1,59 @@
   # First unpack MIME containers and reject serious errors.
-  deny  message		= This message contains a MIME error ($demime_reason)
-        demime		= *
-        condition	= ${if >{$demime_errorlevel}{2}{1}{0}}
+  #
+  deny  message   = This message contains a MIME error ($demime_reason)
+        demime    = *
+        condition = ${if >{$demime_errorlevel}{2}{1}{0}}
         
   # Reject typically wormish file extensions. There is almost no
   # sense in sending such files by email.
   # Note that you might want to change the list of extensions.
+  #
   deny  message = This domain has a policy of not accepting certain types of attachments \
                   in mail as they may contain a virus.  This mail has a file with a .$found_extension \
                   attachment and is not accepted.  If you have a legitimate need to send \
                   this particular attachment, send it in a compressed archive, and it will \
                   then be forwarded to the recipient.
-        demime = ade:adep:adp:bas:bat:chm:cmd:cnf:com:cpl:crt:dll:hlp:hta:inf:ins:isp:js:jse:lnk:mad:maf:mag:mam:maq:mar:mas:mat:mav:maw:ocx:pcd:pif:reg:scf:scr:sct:vbe:vbs:wsc:wsf:wsh:url:xnk
+        demime  = ade:adep:adp:bas:bat:chm:cmd:cnf:com:cpl:crt:dll:hlp:hta:inf:ins:isp:js:jse:lnk:mad:maf:mag:mam:maq:mar:mas:mat:mav:maw:ocx:pcd:pif:reg:scf:scr:sct:vbe:vbs:wsc:wsf:wsh:url:xnk
 
   # Reject virus infested messages.
-  warn  message		= This message contains malware ($malware_name)
-        malware		= *
-        log_message	= This message contains malware ($malware_name)
+  #
+  warn    message     = This message contains malware ($malware_name)
+          malware     = *
+          log_message = This message contains malware ($malware_name)
+
+  # Reject messages with an "X-Spam-Flag: YES" header.
+  #
+  deny    message = Sending server openly considers message as spam ('$regex_match_string' found).
+          regex   = ^X-Spam-Flag:: *YES
```

Comment:
There's one problem with this: it might match body as well. But I don't think this is a big issue.